tests/app/main/views/test_users.py

```diff
@@ -245,6 +245,8 @@ def _assert_things_about_user_export(self, response, users):
             assert sorted(list(users[0].keys())) == sorted(rows[0])
 
             for index, user in enumerate(users):
+                print("LHS: {}".format(sorted([six.text_type(val) for val in user.values()])))
```

Comment:
No that was some debug stuff I forgot to remove - they do print out if the test fails, I'll drop them now.